File: hackerrank/Code Actually Day 1.py
t=int(input())
while(t):
    n=int(input())
    sum1=0
    avg=0
    count=0
    a = list(map(int,input().split()))[:n]
    for i in range(n):
        sum1 =sum1 +a[i]
    avg=sum1/n
    for i in range(n):
        if(a[i]<avg):
            count +=1
    print(count) 
    t -=1

File: hackerrank/Code Actually Day 1.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    int t;
    cin>>t;
    while(t--){
        int n,count=0;
        float avg=0 ,sum=0;
        cin>>n;
        int m[n];
        for(int i=0;i<n;i++){
            cin>>m[i];
            sum += m[i];
        }
        avg=sum/n;
        for(int i=0;i<n;i++){
            if(m[i]<avg){
                count++;
            }
        }
        cout<<count<<"\n";
    }
    return 0;
}
